src/Rover.cpp

```diff
@@ -16,31 +17,64 @@
 extern "C"
 {
     #include "HindsightCAN/CANMotorUnit.h"
+    #include "HindsightCAN/CANSerialNumbers.h"
+    #include "HindsightCAN/CANCommon.h"
+}
+
+void initEncoders()
+{
+    CANPacket p;
+    for (uint8_t serial = DEVICE_SERIAL_MOTOR_BASE;
+        serial < DEVICE_SERIAL_MOTOR_HAND; // The hand motor doesn't have an encoder
+        serial ++ ) {
+      AssembleEncoderInitializePacket(&p, DEVICE_GROUP_MOTOR_CONTROL, serial,
+          0, 0, 1); // 1 means to zero the encoder angle measurement
+      sendCANPacket(p);
+      usleep(1000); // We're running out of CAN buffer space
+      AssembleEncoderPPJRSetPacket(   &p, DEVICE_GROUP_MOTOR_CONTROL, serial,
+          1024); // I have no idea how many pulses actually make one rotation
+      sendCANPacket(p);
+      usleep(1000); // We're running out of CAN buffer space
+      AssembleTelemetryTimingPacket(  &p, DEVICE_GROUP_MOTOR_CONTROL, serial,
+          PACKET_TELEMETRY_ANG_POSITION, 100); // 100 ms, for 10 hz
+      sendCANPacket(p);
+      usleep(1000); // We're running out of CAN buffer space
+    }
 }
 
 void setArmMode(uint8_t mode)
 {
     // Set all arm motors to given mode
     CANPacket p;
-    for (uint8_t serial = 0x1; serial < 0x8; serial ++ ) {
+    for (uint8_t serial = DEVICE_SERIAL_MOTOR_BASE;
+        serial <= DEVICE_SERIAL_MOTOR_HAND;
+        serial ++ ) {
       AssembleModeSetPacket(&p, DEVICE_GROUP_MOTOR_CONTROL, serial, mode);
       sendCANPacket(p);
+      usleep(1000); // We're running out of CAN buffer space
     }
 }
 
-void InitializeRover()
+void InitializeRover(uint8_t arm_mode)
 {
     InitializeCANSocket();
 
     // Set all wheel motors to mode PWM
     CANPacket p;
     uint8_t mode_PWM = 0x0;
```

Comment:
This is a super minor thing but is this variable `mode_PWM` being used anymore? Noticed it was previously but now a constant `MOTOR_UNIT_MODE_PWM` is being used instead